aminator/plugins/cloud/ec2.py

```diff
@@ -76,8 +76,15 @@ def _retry(f, *args, **kwargs):
                     _tries -= 1
                     _delay *= backoff
                 else:
-                    log.critical("Unable to retry register_image due to ClientError: %s", e)
-                    return False
+                    # This could be a retryable error, such as ec2 api throttling
+                    _tries -= 1
+                    if (tries > 0):
```

Comment:
should this be
```suggestion
                    if (_tries > 0):
```
instead?

Reply:
yep, thanks!